crates/circuit/src/circuit_data.rs

```diff
@@ -473,6 +473,13 @@ impl CircuitData {
         Ok(())
     }
 
+    pub fn qubit_index(&self, qubit: ShareableQubit) -> Option<u32> {
+        self.qubits.find(&qubit).map(|qubit| qubit.0)
+    }
+
+    pub fn clbit_index(&self, clbit: ShareableClbit) -> Option<u32> {
+        self.clbits.find(&clbit).map(|clbit| clbit.0)
+    }
```

Comment:
Did you intend for these to be part of the other, non-`#[pymethods]` `impl CircuitData` block?